addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -123,3 +123,24 @@ test("create sub thread from sub-thread list", async () => {
     await click("button[aria-label='Create Thread']");
     await contains(".o-mail-Discuss-threadName", { value: "MyEpicThread" });
 });
+
+test("create sub thread from another sub thread via the action button", async () => {
```

Comment:
`'Threads' menu also available in a Thread`

Current name of test is decent, this proposition aims at making it more readable and better grasp the intent of test at a glance (also trying to keep same terminology as end-users).